skimage/measure/_regionprops_utils.py

```diff
@@ -326,3 +327,28 @@ def perimeter_crofton(image, directions=4):
 
     total_perimeter = coefs @ h
     return total_perimeter
+
+def check_spacing(spacing, ndims:int):
+    """Checks the spacing parameter
+
+    The `spacing` parameter should be a tuple.
+    If the spacing is a single number (e.g. float)
+    then assume spacing is same in all dimensions.
+
+    Parameters
+    ---------
+    spacing: The spacing of the pixels in the image
+
+    Returns
+    -------
+    Corrected spacing as tuple (if needed)
+    """
+
+    if isinstance(spacing, (tuple, List)):
```

Comment:
I'm usually in the camp: don't make input sanitation overly complicated. So why not call `np.array(spacing)` and be done with it? This also makes checking the dimension and type easier.

Also don't forget to check if the length matches `ndim` if the input is a tuple or sequence (whatever we settle on).

Reply:
`np.array` doesn't seem to work. Instead I went with @jni 's recommendation of `np.ndim(spacing) == 1`- which will also handle the case where an np.array of size > 1 is passed.